source/modules/MEI.xml

```diff
@@ -3071,6 +3071,31 @@
       </rng:data>
     </content>
   </macroSpec>
+  <macroSpec ident="data.PGFORMFUNC" module="MEI" type="dt">
```

Comment:
Could you please explain why this is named `data.PGFORMFUNC`? Specifically, why have the "FORM" part in there? If it's necessary, it should also be part of the `<desc>`, I guess?!

Reply:
having spoken to you, this is about the function of the page form. The wording seems a bit poor, and is under revision right now…

Reply:
I renamed the data type to PGFUNC